tests/test_integration_layout.py

```diff
@@ -285,22 +297,26 @@ def on_mount(self):
             self.styles.width = large_widget_size[0]
             self.styles.height = large_widget_size[1]
 
+    container_style_rules = (
+        [container_style] if isinstance(container_style, str) else container_style
+    )
+
     class LargeWidgetContainer(Widget):
+        # TODO: Once textual#581 ("Default versus User CSS") is solved the following CSS should just use the
+        #  "LargeWidgetContainer" selector, without having to use a more specific one to be able to override Widget's CSS:
         CSS = """
-        LargeWidgetContainer {
```

Comment:
It seems that Widget's `CSS` declarations take precedence over children classes' ones - i.e. setting a custom `scrollbar-size-vertical` here is always overridden by the Widget class' own one. Which is why this test broke when the breaking change was introduced.
A temporary workaround (preceded by with a `TODO` comment so we don't forget to remove it later on) is to use a more specific selector in this test :slightly_smiling_face: